README.md

```diff
@@ -1,54 +1,61 @@
-# Home
+<h2 align="center">Home</h2>
 
-“Home” - is an all-in-one social service that will cover all aspects of your communication with your home and neighbors.
-It includes next modules: 
-- Communication with residents of the house, notification
-- Suggestions / wishes residents of the house
-- Bots for social chats and networks
-- Reporting system
-- Voting system
-- Communal payments
-- OSBB module
-- Volunteer module to help people with disabilities 
-- Parking module
+[![Home](https://www.chalets-usa.co.uk/res/Ascent%20Beaver%20Creek%20Colorado%20USA%20self-catered%20apartments.jpg)](https://github.com/ita-social-projects/Home)
+___
 
 [![Heroku Deploy](https://img.shields.io/website?down_color=red&down_message=heroku%20down&up_color=green&up_message=heroku%20up&url=https%3A%2F%2Fhome-project-academy.herokuapp.com%2Fapi%2F0%2Fapidocs%2Findex.html)](https://home-project-academy.herokuapp.com/api/0/apidocs/index.html)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=ita-social-projects-home&metric=alert_status)](https://sonarcloud.io/dashboard?id=ita-social-projects-home)
 [![Coverage Status](https://img.shields.io/sonar/coverage/ita-social-projects-home?server=https%3A%2F%2Fsonarcloud.io)](https://sonarcloud.io/component_measures?id=ita-social-projects-home&metric=coverage&view=treemap)
 [![Github Issues](https://img.shields.io/github/issues/ita-social-projects/Home?style=flat-square)](https://github.com/ita-social-projects/Home/issues)
 [![Pending Pull-Requests](https://img.shields.io/github/issues-pr/ita-social-projects/Home?style=flat-square)](https://github.com/ita-social-projects/Home/pulls)
+[![Docker Pulls](https://img.shields.io/docker/pulls/homeacademy/home-application)](https://hub.docker.com/r/homeacademy/home-application)
 [![License](http://img.shields.io/:license-mit-blue.svg?style=flat-square)](http://badges.mit-license.org)
 
----
+__Home__ - is an all-in-one social service that will cover all aspects of your communication with your home and
+neighbors.
+
+It includes next modules:
+- Communication with residents of the house, notification
+- Suggestions / wishes residents of the house
+- Bots for social chats and networks
+- Reporting system
+- Voting system
+- Communal payments
+- OSBB module
+- Volunteer module to help people with disabilities
+- Parking module
+___
 
 ## Table of Contents
 
 - [About the project](#About-the-project)
 - [Installation](#installation)
   - [Required to install](#Required-to-install)
   - [Environment](#Environment)
-  - [Clone](#Clone)
-  - [Setup](#Setup)
   - [How to run local](#How-to-run-local)
+  - [Docker images](#Docker-images)
 - [Documentation](#Documentation)
 - [Contributing](#contributing)
   - [git flow](#git-flow)
   - [issue flow](#git-flow)
 - [FAQ](#faq)
```

Comment:
There is no chapter "FAQ" but instead is chapter "Team". What if to change "FAQ" by "Team"?